libmate-desktop/mate-colorsel.c

```diff
@@ -1162,19 +1167,18 @@ palette_change_color (GtkWidget         *drawing_area,
 {
   gint x, y;
   MateColorSelectionPrivate *priv;
-  GdkColor gdk_color;
-  GdkColor *current_colors;
+  GdkRGBA gdk_color;
+  GdkRGBA *current_colors;
   GdkScreen *screen;
 
   g_return_if_fail (MATE_IS_COLOR_SELECTION (colorsel));
   g_return_if_fail (GTK_IS_DRAWING_AREA (drawing_area));
 
   priv = colorsel->private_data;
 
-  gdk_color.red = UNSCALE (color[0]);
-  gdk_color.green = UNSCALE (color[1]);
-  gdk_color.blue = UNSCALE (color[2]);
-  gdk_color.pixel = 0;
+  gdk_color.red = color[0];
+  gdk_color.green = color[1];
+  gdk_color.blue = color[2];
```

Comment:
do you need to initialize `gdk_color.alpha`?

Reply:
No need to initialize ```alpha```, this value is not needed
The ```gdk_color``` is ultimately used by ```mate_color_selection_palette_to_string``` and does not require parsing of alpha